integration-tests/src/tests/build_spec.rs

```diff
@@ -40,7 +40,7 @@ fn build_spec_testing_node() {
 	assert_eq!(
 		output.status.success(),
 		true,
-		"Make sure you compile the polimec-parachain-node with \"--release\" and \"--features std,fast-gov\" before running this test."
+		"Make sure you compile the polimec-parachain-node with \"--release\" and \"--features std,fast-mode\" before running this test."
```

Comment:
I get:

> error: none of the selected packages contains these features: fast-mode

Should this be: 

```suggestion
		"Make sure you compile the polimec-parachain-node with \"--release\" and \"--features std,instant-mode\" before running this test."
```
?

P.S: Even with `cargo t --release --features std,instant-mode` the test is failing

Reply:
Hmmm which command are you actually running? We added the default-members so everything that does not have a clear specified -p will only use `nodes/*` and `pallets/*` so this test shouldn't be running if we call cargo test. Tbh I do not think this test should be in the integrations test folder as it requires the node build with specific features, that are now also different from the default integration test features. Might be better to move it to some new package like "UI testing" that is specifically build to do Multi project instantiations for front-end testing. 

But I can try to fix the test and feature flags. Wdyt @JuaniRios ?

Reply:
We can build the node with instant or fast mode, it doesn't matter.
But it's a valid point that this test might not belong there if it requires some previous step in order to pass.

Leo's problem was that he forgot to compile the node before running that test.

We discussed also maybe adding a #[ignore], and then running it manually if we ever want to, although would be nice to have it automated so we never forget. Open to any other suggestions

Reply:
I think it would be nice to create a separate folder for create multi project nodes etc where we have a specific Readme + this test + maybe all the necessary cargo requirements.

Reply:
For now we can do an #[ignore]

Reply:
`cargo b -p polimec-parachain-node --release --features std,fast-mode && cd integration-tests && cargo t build_spec_testing_node -- --nocapture`

> Members length cannot exceed MaxMembers.

![image](https://github.com/Polimec/polimec-node/assets/4128940/c8b477f2-5276-417d-9a22-95063c5da611)

I think it's because we set 6 members in the technical council while the maximum is 5.

Reply:
Interesting. Thanks I'll take a look!

Reply:
Okay fixed. Was indeed max 5 technical committee problem.

Reply:
Tested with `cargo t build_spec_testing_node -- --include-ignored`. It works!